onnx/onnxifi.h

```diff
@@ -289,6 +301,35 @@ typedef int32_t onnxBackendInfo;
  */
 #define ONNXIFI_BACKEND_DEVICE_TYPE 6
 
+/**
+ * List of supported ONNX IR versions.
+ *
+ * Value type: char[], e.g.:
+ *    "3" (IR version in ONNX 1.0)
+ *
+ * Possible values: space-separated list of supported ONNX IR versions,
+ *     represented as decimal integers. ONNX IR versions must match values
+ *     in ONNX Version enum.
+ */
+#define ONNXIFI_BACKEND_ONNX_IR_VERSION 7
```

Comment:
ir version is not tied to onnxifi nor "backend", so probably better to just call it ONNX_IR_VERSION

Reply:
Backend is a combination of software + hardware, so different backends may support different IR versions. This is why the constant has `BACKEND` in its name.